app/representers/service_representer.rb

```diff
@@ -25,6 +25,7 @@ module ServiceRepresenter
   property :buyer_can_select_plan
   property :buyer_plan_change_permission
   property :notification_settings
+  property :annotations_hash, as: :annotations
```

Comment:
I wondder if we can include something into these representers to avoid adding this line everywhere. But it's just one line, so maybe better than inventing something more complicated.

Reply:
I'll probably change the format, like you suggest in the other comment, so maybe this is not needed anymore